java/spi.java.hints/src/org/netbeans/modules/java/hints/spiimpl/Utilities.java

```diff
@@ -1452,6 +1452,7 @@ protected JCExpression checkExprStat(JCExpression t) {
             return super.checkExprStat(t);
         }
 
+/*  TODO: unused? called a method which is only available from 8-11.
         protected JCCase switchBlockStatementGroupImpl() throws Throwable {
```

Comment:
i commented it out, but I think we can remove this, right?

Reply:
Yes, I think so.

Reply:
All tests green with the requested changes integrated. 

Lets try to remove the ByteBuddy interceptor which delegates to one of the two methods.

Reply:
interceptor removal is squashed with the trees commit 748afc8